tests/unittests/CoreGraphics/CGFontTests.mm

```diff
@@ -106,4 +106,38 @@
     CGFontRef font = CGFontCreateWithFontName(c_arialBoldItalicName);
     CFAutorelease(font);
     EXPECT_EQ(-434, CGFontGetDescent(font));
+}
+
+TEST(CGFont, CreateWithDataProvider) {
+    // get test startup full path
+    wchar_t fullPath[_MAX_PATH];
+    size_t len = GetModuleFileNameW(NULL, fullPath, _MAX_PATH);
+
+// OSX has twice-as-wide wchars
+#if TARGET_OS_WIN32
+    NSString* testFileFullPath = [NSString stringWithCharacters:(const unichar*)fullPath length:len + 1];
+#else
+    NSString* testFileFullPath = [NSString stringWithBytes:fullPath length:sizeof(wchar_t) * len encoding:WCHAR_ENCODING];
+#endif
+
+    // reconstruct fullpath for test artifact file. e.g., /Volumes/WinObjC/build/Tests/UnitTests/Foundation/OSX/data/NSFileManagerUT.txt
+    testFileFullPath = [[testFileFullPath stringByDeletingLastPathComponent] stringByAppendingString:@"/data/WinObjC-Regular.ttf"];
```

Comment:
done but it ended up requiring a much broader change than I think you intended :(

---
In reply to: [88291272](https://github.com/Microsoft/WinObjC/pull/1386#discussion_r88291272) [](ancestors = 88291272,88152992,88152656)